features/apply.feature

```diff
@@ -1,33 +1,33 @@
 Feature: User can Apply to CMR
   Scenario: User Successfully applies to Contra Costa
-    Given that "apply/" loads
+    Given that "/apply/" loads
```

Comment:
I realized we had inconsistent and fragile ways of checking URLs, so I tried to ensure they all included the first slash and that comparisons were more able to handle deviations.